src/main/java/de/wejago/hichi2influx/repository/InfluxDbRepository.java

```diff
@@ -0,0 +1,50 @@
+package de.wejago.hichi2influx.repository;
+
+import com.influxdb.client.InfluxDBClient;
+import com.influxdb.client.WriteApiBlocking;
+import com.influxdb.client.domain.WritePrecision;
+import com.influxdb.client.write.Point;
+import de.wejago.hichi2influx.config.InfluxDBConfig;
+import de.wejago.hichi2influx.dto.SensorEntry;
+import java.time.Instant;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@RequiredArgsConstructor
+public class InfluxDbRepository {
+    private final InfluxDBConfig influxDBConfig;
+
+    private InfluxDBClient influxDBClient;
+
+    public Point writePoint(SensorEntry sensorEntry) {
+        setConnectionIfNeeded();
+        WriteApiBlocking writeApi = influxDBClient.getWriteApiBlocking();
```

Comment:
Does that need to be closed afterwards?

Reply:
If we close it then we won't be able to use it again for the next write. So we can remove the `updateConnectionIfNeeded()` because it will be always needed.

Reply:
I mean close/release the writeApi

Reply:
I could not find close() method for the `writeApi`. In the documentation and the sample project they are not closing it either.

Reply:
OK